CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -401,16 +437,22 @@ ally will have to make a skill check to swap back their muzzle attachment.
 Build Tree
 ==============================
 Note:
-Nanites are used to "glue" modular bits of tech together and to allow tech to function. After a certain amount of time the Nanites die and the item crumbles. At higher levels, the Nanites are able to last longer so the item can be used longer. 
-Drones do not count as "cybernetic allies" for most repair skills, though you can repair them with
-repair kits as normal. They take their turns independently, with 4 actions per turn, and they
-take their owner's (not necessarily their deployer's) initiative. A Drone can only use 4 IO ports 
-at a time unless otherwise stated. 
-Unlocking a Build tree skill requires having unlocked the skill below it on the 
-tree. Please see the Engineer Skill Tree - Build.png for dependencies.
+    Nanites are used to "glue" modular bits of tech together and to allow tech 
```

Comment:
>Nanites are used to "glue" modular bits of tech together and to allow tech 

Should there be a blank new-line before this?

Reply:
Doesn't have to be, the section heading is terminated on the colon. That said, the section 'Note' doesn't really add any information for the reader. I think it looks cleaner if I delete it.